esp-box/include/bsp/esp-box.h

```diff
@@ -274,9 +274,12 @@ esp_err_t bsp_spiffs_unmount(void);
 #define BSP_LCD_PIXEL_CLOCK_HZ     (40 * 1000 * 1000)
 #define BSP_LCD_SPI_NUM            (SPI3_HOST)
 
-#define BSP_LCD_DRAW_BUFF_SIZE     (BSP_LCD_H_RES * 10)
 #define BSP_LCD_DRAW_BUFF_DOUBLE   (0)
 
+#ifndef BSP_LCD_DRAW_BUFF_SIZE
```

Comment:
Just out of curiosity, why is it not a value which is set at run time? We aren't allocating this buffer statically anyway, right? (Or, if making it a compile time setting, why not via a Kconfig option?)

Reply:
I am in favor of the first solution -> runtime settings.

The initial API was drafted with an idea of as simple as possible functions, often with no parameters, so the user could just call a few functions to initialize his development board. The requirement has changed now, as we want to ship BSP with various GUI projects. Let's now introduce a breaking change, that will be open for extensions and closed for modifications :)

Reply:
@tore-espressif Ok, I am working on it. Thank you